README.md

```diff
@@ -53,8 +53,39 @@ and analyze all the data included in the various ‘many packages’. For
 example, the `manydata::get_packages()` function can be used to discover
 the packages currently available in the many packages universe.
 
-    # remotes::install_github("globalgov/manydata")
-    manydata::get_packages()
+``` r
+manydata::get_packages()
+```
+
+
+    ## # A tibble: 7 × 6
+    ##   name        full_name            
+    ##   <chr>       <chr>                
+    ## 1 manydata    globalgov/manydata   
+    ## 2 manyenviron globalgov/manyenviron
+    ## 3 manyhealth  globalgov/manyhealth 
+    ## 4 manypkgs    globalgov/manypkgs   
+    ## 5 manystates  globalgov/manystates 
+    ## 6 manytrade   globalgov/manytrade  
+    ## 7 messydates  globalgov/messydates 
+    ##   description                                                          
+    ##   <chr>                                                                
+    ## 1 An R portal for ensembled global governance data                     
+    ## 2 R Package for ensembled data on environmental agreements             
+    ## 3 An R package for ensembled data on international health organisations
+    ## 4 Support for creating new manyverse packages                          
+    ## 5 An R package for ensembled data on sovereign states                  
+    ## 6 An R package for ensembled data on trade agreements                  
+    ## 7 An R package for ISO's Extended Date/Time Format (EDTF)              
+    ##   installed latest        updated   
+    ##   <chr>     <chr>         <date>    
+    ## 1 0.7.4     0.7.4         2022-06-03
+    ## 2 0.1.2     0.1.2         2022-03-16
+    ## 3 <NA>      0.1.1-272ea19 2022-02-15
+    ## 4 0.2.1     0.2.1         2022-02-18
+    ## 5 0.1.0     0.0.6         2021-12-06
+    ## 6 0.1.2     0.1.1         2022-03-23
+    ## 7 0.3.0     0.2.1         2022-02-18
```

Comment:
A pity this runs on multiple lines. Maybe reorder so `description` comes at the end?